dev-tools/export_dashboards.py

```diff
@@ -119,7 +115,7 @@ def main():
     args = parser.parse_args()
 
     if args.index is None:
-        args.index = args.beat.lower() + "-*"
+        args.index = args.beat.lower()
```

Comment:
@monicasarbu Could you have a quick look at this one just to make sure this is as expected. It seems all the new index patterns id's don't have `-*` anymore.